scripts/compute_resources.py

```diff
@@ -12,7 +12,7 @@
 def get_resource_usage():
     try:
         result = subprocess.run(
-            "make tests",
+            "make tests-v0",
```

Comment:
so right now we don't run the v1 in the ci ?

Reply:
no right now it's only ran on v0. The main reason for this is that I need to add a CI in ssj to compile the contracts and save them as artifacts.